Library/Homebrew/test/cask/installer_spec.rb

```diff
@@ -3,10 +3,6 @@
 
 describe Cask::Installer, :cask do
   describe "install" do
-    let(:empty_depends_on_stub) {
```

Comment:
Appears to be unused (the last reference was removed in `7d5b8a5cea122c09f297dffa6b76187c413a9565`)